### ISSUE TITLE
Node.js standards-positions

### ISSUE THREAD

**Issue by jasnell** [whatwg/meta#304]
Just a heads up... the Node.js project is finalizing a proposal to establish a new nodejs/standards-positions repository that we will use to discuss and publish consensus positions on various proposed standards, specs, etc. See https://github.com/nodejs/admin/issues/841 for details. Assuming the proposal goes through I will be creating the new repo next week. If any of the WHATWG activity areas that are relevant to Node.js (streams, url, fetch, encoding, etc) wish to seek out the projects official position on any proposed specification change, that repo will become the ideal location to do so. I will update here once we have things set up.

/cc @domenic @annevk

**Comment by annevk** [comment]
WHATWG typically doesn't do formal position seeking in this matter.

We look for agreement among implementers and no dissent. See https://whatwg.org/working-mode. In practice this often comes down to pinging some people, as we have been doing.

**Comment by jasnell** [comment]
Yeah, that obviously works as well and is largely what I expected, but wanted to make sure to give y'all a heads up on it in case you saw this mentioned or it came up in conversations.
